ecosystem/models/configuration.py

```diff
@@ -145,27 +138,18 @@ def __repr__(self):
         return pprint.pformat(self.to_dict(), indent=4)
 
 
+@dataclass
 class PythonRepositoryConfiguration(RepositoryConfiguration):
-    """Repository configuration for python based projects."""
+    """
+    Repository configuration for python based projects.
+    """
 
-    def __init__(
-        self,
-        language: Optional[LanguageConfiguration] = None,
-        dependencies_files: Optional[List[str]] = None,
-        extra_dependencies: Optional[List[str]] = None,
-        tests_command: Optional[List[str]] = None,
-        styles_check_command: Optional[List[str]] = None,
-        coverages_check_command: Optional[List[str]] = None,
-    ):
-        language = language or PythonLanguageConfiguration()
-        super().__init__(
-            language=language,
-            dependencies_files=dependencies_files,
-            extra_dependencies=extra_dependencies,
-            tests_command=tests_command,
-            styles_check_command=styles_check_command,
-            coverages_check_command=coverages_check_command,
-        )
+    _tox_template: Template = None
```

Comment:
These type hints are lies. They should be `Template | None` because you assign it to `None`. You need `from __future__ import annotations` for this syntax.